chainstate/src/interface/chainstate_interface_impl.rs

```diff
@@ -286,6 +286,28 @@ impl<S: BlockchainStorage, V: TransactionVerificationStrategy> ChainstateInterfa
             .map_err(ChainstateError::FailedToReadProperty)
     }
 
+    fn last_common_block(
```

Comment:
last_common_ancestor

Reply:
We already have `last_common_ancestor`:
```
    fn last_common_ancestor(
        &self,
        first_block_index: &GenBlockIndex,
        second_block_index: &GenBlockIndex,
    ) -> Result<GenBlockIndex, ChainstateError> {
```
(I can't use it directly because we don't have `GenBlockIndex` in the wallet controller).

Reply:
last_common_ancestor_by_id?